packages/gafl-webapp-service/src/pages/summary/contact-summary/route.js

```diff
@@ -101,7 +112,11 @@ class RowGenerator {
   }
 
   generateContactRow (label, href, visuallyHiddenText, id, contactTextSpec = CONTACT_TEXT_DEFAULT) {
-    return this._generateRow(this.labels[label], this._getContactText(contactTextSpec), href, this.labels[visuallyHiddenText], id)
+    const contactText =
+      label !== 'contact_summary_row_contact'
```

Comment:
Reverse the logic here - it's generally best practice to go for `if (x === y) { /* stuff */ } else { /*other stuff */ }`